modules/core/src/adapter/attribute-utils/get-attribute-from-layouts.ts

```diff
@@ -135,18 +125,29 @@ function getAttributeFromBufferLayout(
     for (const interleavedMapping of bufferMapping.attributes || []) {
       const byteOffset = nextByteOffset;
       nextByteOffset +=
-        interleavedMapping?.byteStrideOffset ||
+        interleavedMapping?.byteOffset ||
```

Comment:
This does not look right. `byteOffset` is an absolute value, not an increment. Also `||` will not account for the case where it is specified as 0.